lib/1.4/dml-builtins.dml

```diff
@@ -2763,7 +2763,7 @@ also used by the default implementations of hard reset, soft reset and
 power-on reset. Defaults to 0.
 </dd></dl>
 */
-template field is (object, _bitsize, _lsb, init_val, shown_desc, _get, _set) {
+template field is (object, _bitsize, _lsb, init_val, shown_desc, _get, _set, _qname) {
```

Comment:
`_qname` being inherited by `field` increases vtable size. How large of a concern is this?

Reply:
We are adding one small method declaration for each field. This is not entirely trivial but we'd have to do some sort of hack to avoid this

Reply:
It's not only vtable, it's also code generation of the _qname() implementation.  And fields are a major bottleneck. I think this is a reason why we haven't done this before.

So, I think this fix needs a more elaborate study of what it costs. Try to find the worst real example of a slow compilation with too many fields, and see how much slower it gets.